src/applications/vaos/services/appointment/transformers.js

```diff
@@ -538,7 +540,9 @@ export function transformConfirmedAppointments(appointments) {
       minutesDuration,
       comment:
         appt.instructionsToVeteran ||
-        appt.vdsAppointments?.[0]?.bookingNote ||
+        (appt.communityCare
+          ? 'vistaCCAppt'
```

Comment:
I think this will put `vistaCCAppt` in the instructions field for regular CC appointments that don't have instructions, which I don't think is what we want. We might want to break this into a separate function to be able to write the logic more clearly.

That said, I'm not positive we want to put a special value in the instructions here, since I don't think the design calls for putting the text in the right hand column like other instructions. I would check with Lenae on that.